src/Strategy/Python/SetupPy.hs

```diff
@@ -45,8 +46,11 @@ type Parser = Parsec Void Text
 installRequiresParser :: Parser [Req]
 installRequiresParser = prefix *> entries <* end
   where
-  prefix  = skipManyTill anySingle (string "install_requires=[")
-  entries = between quote quote requirementParser `sepBy` char ','
-  end     = char ']'
+  prefix  = skipManyTill anySingle (symbol "install_requires") *> symbol "=" *> symbol "["
+  entries = (between quote quote requirementParser) `sepEndBy` symbol ","
```

Comment:
How would this `between quote quote ...` handle mismatched quoting styles like `install_requires = ["requests']`?
Open question: Should it be permissive and accept mismatches, or should it just search for matching quotes?

Reply:
You're right; fixed: https://github.com/fossas/spectrometer/pull/119/commits/f778e9fd4125cc50ea936ff622e368bc0648015f